<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FingerprintSimilarityDialog</class>
 <widget class="ChemData::AbstractClusteringWidget" name="FingerprintSimilarityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fingerprint Similarity</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="controlsLayout">
       <item>
        <widget class="QLabel" name="fingerprintLabel">
         <property name="text">
          <string>Fingerprint:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fingerprintComboBox">
         <item>
          <property name="text">
           <string>FP2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PubChem</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="similarityLabel">
         <property name="text">
          <string>Similarity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="similaritySlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="similaritySpinBox">
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChemData::AbstractClusteringWidget</class>
   <extends>QWidget</extends>
   <header location="global">chemdata/gui/abstractclusteringwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
